source/common/runtime/runtime_features.cc

```diff
@@ -89,6 +89,7 @@ constexpr const char* runtime_features[] = {
     "envoy.reloadable_features.unify_grpc_handling",
     "envoy.reloadable_features.upstream_http2_flood_checks",
     "envoy.restart_features.use_apple_api_for_dns_lookups",
+    "envoy.reloadable_features.header_map_coalesce_cookie_headers",
```

Comment:
I don't think this will result in cookies coalescing, but in correctly coalescing cookies.  WDYT?

Reply:
renamed it!